src/main/java/bibliotecame/back/Extension/ExtensionController.java

```diff
@@ -1,5 +1,6 @@
 package bibliotecame.back.Extension;
 
+import bibliotecame.back.ErrorMessage;
```

Comment:
I leave my comment here cause the related lines were not modified.

Note that there are some endpoints that return the response of the `ExtensionService` but some method are retuning the plain error message instead of json (for example `createExtension` method).

Look for all the places in the app (not only this file) where this is happening and fix it.